apps/strava/strava.star

```diff
@@ -138,11 +136,6 @@ def main(config):
     units = config.get("units", DEFAULT_UNITS)
     display_type = config.get("display_type", DEFAULT_SCREEN)
 
-    if refresh_token and is_rate_limited():
```

Comment:
We now support 429s natively in the HTTP client. I've updated the responses to check for a 429 to mimic this behavior.